teamwork/templates/projects/view_project.html

```diff
@@ -739,7 +761,7 @@ <h3 class="box-title">Team Chat</h3>
     <div class="box-header with-border">
         <h6 class="box-title">Pending Requests to Join: ( {{ pending_count }} )</h6>
         <div class="box-tools">
-            <button type="button" class="btn btn-box-tool" data-widget="collapse"><i class="fa fa-minus"></i></button>
+            <button type="button" class="btn btn-box-tool" data-widget="collapse"><i class="fa fa-plus"></i></button>
```

Comment:
Noice

Reply:
Lol you started with a minus for some reason when the box was already closed